### DISCUSSION TITLE
External db  - link to another record [May 12]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2086] [Discord Threads]
> 🙋‍♂️ **eddowding**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### * which database are you using   
#### * if its connecting to external db - it should automatically identify link to another record  
#  
> 🙋‍♂️ **eddowding**  
#### mysql; and .. it's not 😦  
#  
> 💁 **o1lab**  
#### what version of mysql is it ?  
#  
> 💁 **o1lab**  
#### possible for a quick call  
#  
> 🙋‍♂️ **eddowding**  
#### yep .. 5 mins  
#  
> 💁 **o1lab**  
#### sure will DM a zoom link  
#  
> 🙋‍♂️ **eddowding**  
#### thanks!  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/973945213850640404)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
